routes/reviewRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+
+const { verifyJWT } = require("../middleware/auth");
+const reviewController = require("../controllers/reviewController");
+
+const router = express.Router();
+
+router.post("/:productId", verifyJWT, reviewController.reviewSignup);
```

Comment:
```suggestion
router.post("/:productId", verifyJWT, reviewController.createReview);
```
reviewSignup이라는 네이밍 보다는 CRUD에 맞춰 create 혹은 register로 쓰이는게 더 명확해 보입니다.